agent/packaging/ubuntu_xenial/kontena-agent/lib/systemd/system/docker.service.d/50-kontena.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment='DOCKER_OPTS=--insecure-registry="10.81.0.0/19"'
```

Comment:
annoying :/ the only way is then to override the ExecStart. any idea if there's some easy way to grab the current ExecStart and dynamically add the `DOCKER_OPTS` into it? Not sure if we would want to just do plain override.